### PR TITLE
[AMDGPU] Add gfx1150 SALU Float instructions

### PR DESCRIPTION
**Pull request by mbrkusanin**

### CODE REVIEW

**Comment by jayfoad** [comment]
This is just the MC layer (assembler/disassembler) support, right? LGTM but we should allow some time for others to review it.

**Comment by mbrkusanin** [comment]
> This is just the MC layer (assembler/disassembler) support, right? LGTM but we should allow some time for others to review it.

Correct. https://github.com/llvm/llvm-project/pull/66885 contains codegen parts.

**Review by rampitec — APPROVED** [review]

File: llvm/test/MC/Disassembler/AMDGPU/gfx1150_asm_salu_float.txt [hunk 1]
```diff
@@ -0,0 +1,2527 @@
+# RUN: llvm-mc -arch=amdgcn -mcpu=gfx1150 -disassemble -show-encoding < %s | FileCheck -check-prefixes=GFX1150 %s
```
Comment (jayfoad) [inline llvm/test/MC/Disassembler/AMDGPU/gfx1150_asm_salu_float.txt:1]: It looks like this was added with the wrong file name, `_asm_` instead of `_dasm_`?

Comment (mbrkusanin) [reply llvm/test/MC/Disassembler/AMDGPU/gfx1150_asm_salu_float.txt:1]: https://github.com/llvm/llvm-project/commit/0219bd3992a1b9c0ae254e3884c6da7880472fa6

Comment (jayfoad) [reply llvm/test/MC/Disassembler/AMDGPU/gfx1150_asm_salu_float.txt:1]: Thanks!

**Pull request merged**